crates/astria-cli/src/commands/rollup.rs

```diff
@@ -0,0 +1,370 @@
+use std::{
+    env::{
+        self,
+        consts::OS,
+    },
+    fs::File,
+    io::{
+        Read,
+        Write,
+    },
+    path::PathBuf,
+    process::Command,
+};
+
+use color_eyre::{
+    eyre,
+    eyre::Context,
+};
+
+use crate::{
+    cli::rollup::{
+        ConfigCreateArgs,
+        ConfigDeleteArgs,
+        ConfigEditArgs,
+        DeploymentCreateArgs,
+        DeploymentDeleteArgs,
+    },
+    types::Rollup,
+};
+
+const EVM_ROLLUP_CHART_URL: &str =
+    "https://astriaorg.github.io/dev-cluster/astria-evm-rollup-0.3.1.tgz";
```

Comment:
maybe make this the default and allow the version number to be replaced by some CLI arg? not sure if that makes sense in this context, will we ever have different version of the helm chart?